src/main/java/at/hannibal2/skyhanni/features/mining/eventtracker/MiningEventTracker.kt

```diff
@@ -88,10 +89,21 @@ class MiningEventTracker {
         }
     }
 
+    @SubscribeEvent
+    fun onTick(event: LorenzTickEvent) {
+        if (!event.repeatSeconds(1)) return
+        if (!config.enabled) return
+        if (!LorenzUtils.inSkyBlock || (!config.outsideMining && !LocationUtils.inAdvancedMiningIsland())) return
```

Comment:
Why is isEnabled() not used/usable here?

Reply:
Not here because this is the only time those two lines are used together every other check for enabled is checking if it can send which uses a slightly different logic